fastmri/models/varnet.py

```diff
@@ -141,6 +141,7 @@ def __init__(
         in_chans: int = 2,
         out_chans: int = 2,
         drop_prob: float = 0.0,
+        num_sens_lines: Optional[int] = None,
```

Comment:
nit: we can spare one more character. (don't forget to update elsewhere)
```suggestion
        num_sense_lines: Optional[int] = None,
```